lib/matplotlib/tests/test_image.py

```diff
@@ -441,6 +441,11 @@ def test_zoom_and_clip_upper_origin():
     ax.set_xlim(-0.5, 2.0)
 
 
+def test_nonuniformimage_setcmap():
```

Comment:
needs a `@cleanup` in order to prevent inadvertent side-effects for other tests.